### PR TITLE
Windows installer refresh

### PR DESCRIPTION
**Pull request by wli3**
According to https://github.com/dotnet/cli/issues/6683

windows-summer-installer-update0 1
windows-summer-installer-update1 195
windows-summer-installer-update
Uninstall:
windows-summer-installer-update_uninstall

### CODE REVIEW

**Comment by livarcocc** [comment]
- "The following was installed at..." should be at the same indentation as "The installation was successful"
- .NET Core telemetry does not have a bullet point.

Also, this looks great. Awesome job!

File: packaging/windows/clisdk/bundle.wxl [hunk 1]
```diff
@@ -50,16 +47,30 @@ Ready? Set? Let's go!</String>
...
 
-The data collected is anonymous and will be published in an aggregated form for use by both Microsoft and community engineers under the Creative Commons Attribution License.
+Resources
 
-The .NET Core Tools telemetry feature is enabled by default. You can opt-out of the telemetry feature by setting an environment variable DOTNET_CLI_TELEMETRY_OPTOUT (for example, 'export' on macOS/Linux, 'set' on Windows) to true (for example, 'true', 1). You can read more about .NET Core tools telemetry at https://aka.ms/dotnet-cli-telemetry.
+    • Core Documentation https://aka.ms/dotnet-docs
+    • SDK Documentation https://aka.ms/dotnet-cli-docs
+    • Release Notes https://aka.ms/20-p2-rel-notes
+    • Tutorials https://aka.ms/dotnet-tutorials
+    .NET Core Telemetry https://aka.ms/dotnet-cli-telemetry</String>
```
Comment (livarcocc) [inline packaging/windows/clisdk/bundle.wxl]: Missing bullet here?

File: packaging/windows/clisdk/bundle.wxl [hunk 2]
```diff
@@ -50,16 +47,30 @@ Ready? Set? Let's go!</String>
...
   <String Id="FilesInUseCancelButton">&amp;Cancel</String>
-  <String Id="FirstTimeWelcomeMessage">Installation completed successfully. Welcome to .NET Core!
-
-Learn more about .NET Core at https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.
-
-.NET Core Tools Telemetry
+  <String Id="FirstTimeWelcomeMessage">The installation was successful
 
-The .NET Core Tools include a telemetry feature that collects usage information. It is important that the .NET Team understands how the tools are being used so that we can improve them.
+    The following were installed at [DOTNETHOME]
```
Comment (livarcocc) [inline packaging/windows/clisdk/bundle.wxl]: This should be at the same indentation as "The installation was successful"

**Comment by wli3** [comment]
image

File: packaging/windows/clisdk/bundle.wxl (hunk 1 above)
Comment (wli3) [inline packaging/windows/clisdk/bundle.wxl]: Thanks! it is fixed

File: packaging/windows/clisdk/bundle.wxl (hunk 2 above)
Comment (wli3) [inline packaging/windows/clisdk/bundle.wxl]: fixed

**Comment by omajid** [comment]
Should "ASP.NET Runtime Package Store" be called "ASP.NET Core Runtime Package Store"? Or is ASP.NET Core just a more verbose name for ASP.NET?

**Comment by wli3** [comment]
> Should "ASP.NET Runtime Package Store" be called "ASP.NET Core Runtime Package Store"? Or is ASP.NET Core just a more verbose name for ASP.NET?

@leecow

**Comment by leecow** [comment]
@wli3 - The package store is not specific to ASP.NET ([dotnet/designs/issues/8](https://github.com/dotnet/designs/issues/8)) and should be called "Runtime Package Store".

File: packaging/windows/clisdk/bundle.wxl [hunk 3]
```diff
@@ -50,16 +47,30 @@ Ready? Set? Let's go!</String>
...
-Learn more about .NET Core at https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.
-
-.NET Core Tools Telemetry
+  <String Id="FirstTimeWelcomeMessage">The installation was successful
 
-The .NET Core Tools include a telemetry feature that collects usage information. It is important that the .NET Team understands how the tools are being used so that we can improve them.
+The following were installed at [DOTNETHOME]
+    • .NET Core SDK 2.0.0
+    • .NET Core Runtime 2.0.0
+    • ASP.NET Runtime Package Store
```
Comment (leecow) [inline packaging/windows/clisdk/bundle.wxl]: Store is not specific to ASP. Name should be "Runtime Package Store"

Comment (wli3) [inline packaging/windows/clisdk/bundle.wxl]: image

fixed

**Comment by livarcocc** [comment]
@dotnet-bot Test this please

**Comment by wli3** [comment]
@dotnet-bot test Windows_NT x86 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test OSX10.12 x64 Release Build

**Comment by wli3** [comment]
@MattGertz Could you review this change?

**Pull request merged**